controller/volume_controller.go

```diff
@@ -3647,7 +3647,7 @@ func (c *VolumeController) syncPVCRecurringJobLabels(volume *longhorn.Volume) er
 	}
 
 	if !hasSourceLabel {
-		c.logger.Warnf("Ignoring recurring job labels on Volume %v PVC %v due to missing source label", volume.Name, pvc.Name)
+		logrus.Tracef("Ignoring recurring job labels on Volume %v PVC %v due to missing source label", volume.Name, pvc.Name)
```

Comment:
Why not using the existing c.logger?

Reply:
[There is no trace in c.logger.](https://github.com/longhorn/longhorn-manager/blob/7a5ca17682b43e6da430d8e459566c713eb4a29f/vendor/github.com/sirupsen/logrus/logrus.go#L139-L177)

Reply:
Most log messages' levels are corrected. I think we use INFO as default log level. Then, use `c.logger.Debugf` here. WDYT? @c3y1huang @innobead

Reply:
IIRC we still use debug, confirm?

Reply:
@c3y1huang do u know why it doesn't have trace? Duo to a different interface?

Reply:
> @c3y1huang do u know why it doesn't have trace? Duo to a different interface?

Yes, [it got removed due to backward compatibility concerns](https://github.com/sirupsen/logrus/commit/b54cafe5cee93a35bf6b061e261e8265ea0a628c).

Reply:
> IIRC we still use debug, confirm? @derekbit

How about changing it to INFO from v1.5.1?
I've adjusted most of log messages' levels.

Reply:
Updated.